README.md

```diff
@@ -1,8 +1,25 @@
-# awesome-summernote
+# Awesome Summernote
+
+[![Awesome](https://cdn.rawgit.com/sindresorhus/awesome/d7305f38d29fed78fa85652e3a63e154dd8e8829/media/badge.svg)](https://github.com/sindresorhus/awesome)
+[![CCO License](https://img.shields.io/badge/license-CC0-blue.svg?style=plastic "CC0 License")](#contributing-and-license)
+
 A curated list for [summernote](https://github.com/summernote/summernote/) plugins and connectors.
 
 This curated list is on very early stage. So, let's make it together!
 
+## Contents
+- [Plugins](#plugins)
+  - [Syntax](#syntax)
+  - [Emojis](#emojis)
+  - [Special Characters & Icons](#special-characters-&-icons)
+  - [Insert](#insert)
+  - [Misc](#misc)
+- [Connectors](#connectors)
+- [Themes and Skins](#themes-and-skins)
+- [Examples](#examples)
+- [How to add](#how-to-add)
```

Comment:
`#how-to-add` was removed by this commit. (previously, L84-L87)
Do we have to remain them in `contributing.md` ?